components/pedigree/rest/src/main/java/org/phenotips/pedigree/rest/PedigreeConverter.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+package org.phenotips.pedigree.rest;
+
+import javax.ws.rs.*;
+
+import org.phenotips.rest.ParentResource;
+
+import javax.ws.rs.core.MediaType;
+import org.xwiki.rest.resources.RootResource;
+import javax.ws.rs.core.Response;
+
+/**
+ * Resource for converting PED files to JSON with REST
+ *
+ * @version $Id$
+ * @since 1.4m2
+ */
+// Fix annotations
+@Path("/pedigreeConverter")
```

Comment:
I see Patient REST interfaces use a `ParentResource` annotation (e.g. `@ParentResource(RootResource.class)`). It works without it, and I'm not sure if it is a good practice to include it or not (I know we removed it while debugging another problem, maybe we should add it back?). @sdumitriu?